iOS-Email-Client/Libs/Env.swift

```diff
@@ -46,17 +46,26 @@ struct Env {
     }
     
     static var socketURL: String {
+        guard !Env.isProduction else {
+            return "wss://socket.criptext.com:3002"
+        }
         return "wss://socket.criptext.com:3002"
```

Comment:
I think this should be the staging url

Reply:
Gabriel suggested that we do not add staging envs to the repo. This is in case I forget to restore variables before uploading the app to the appstore